### PR TITLE
Add openlink to Customer client

### PR DESCRIPTION
**Pull request by Magnus93**
- [ ] #384 
- [ ] https://github.com/payfunc/model-base/pull/41

### CODE REVIEW

File: Client/Order.ts [hunk 1]
```diff
@@ -35,7 +29,9 @@ export class Order {
 		return await this.connection.get<ModelOrder>(`order/${order}`)
 	}
 	async receipt(order: string, format: "pdf" | "html"): Promise<servly.Response | gracely.Error> {
```
Comment (nilssonemma) [inline Client/Order.ts:31]: Can you add a property called `email?: true` that will if true add `?email=true` as a query to the path?

File: Client/Customer.ts [hunk 2]
```diff
@@ -34,15 +43,23 @@ export class Customer {
 	async viewLink(customer: string, link: string): Promise<string | gracely.Error> {
```
Comment (nilssonemma) [inline Client/Customer.ts:43]: in createLink function, can you add a property called email which if true adds `?email=true` to the path

File: Client/Order.ts (hunk 1 above)
Comment (Magnus93) [reply Client/Order.ts]: Fixed

File: Client/Customer.ts (hunk 2 above)
Comment (Magnus93) [reply Client/Customer.ts:43]: Not needed as far as I can see in neither [customer-create-endpoint](https://github.com/payfunc/backend/blob/master/customer-link-create/index.ts) nor [customer-link-view](https://github.com/payfunc/backend/blob/master/customer-link-view/index.ts)

**Pull request merged**